Travis always fail.

This is the error to fix in order to have Travis working:

``` bash
Ricin@exe/MainWindow.c:126:2: error: unknown type name ‘GListStore’
MainWindow.c:756:4: error: unknown type name ‘GListStore’
MainWindow.c:787:2: error: unknown type name ‘GListStore’
/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:61:37: error: ‘GListModel’ undeclared (first use in this function)
     var friend = (this.friends.get_object (fr.position) as Tox.Friend);

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:61:48: error: expected expression before ‘)’ token
     var friend = (this.friends.get_object (fr.position) as Tox.Friend);

MainWindow.c:1201:4: error: unknown type name ‘GListStore’
MainWindow.c:1205:4: error: unknown type name ‘GListStore’

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:186:40: error: ‘GListModel’ undeclared (first use in this function)
             friend.position = friends.get_n_items ();

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:186:51: error: expected expression before ‘)’ token
             friend.position = friends.get_n_items ();

MainWindow.c:1411:3: error: unknown type name ‘GListStore’
MainWindow.c:1415:3: error: unknown type name ‘GListStore’

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:203:39: error: ‘GListModel’ undeclared (first use in this function)
         friend.position = friends.get_n_items ();

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:203:50: error: expected expression before ‘)’ token
         friend.position = friends.get_n_items ();

MainWindow.c:1601:2: error: unknown type name ‘GListStore’

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:125:37: error: ‘GListModel’ undeclared (first use in this function)
     this.friendlist.bind_model (this.friends, fr => new FriendListRow (fr as Tox.Friend));

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:125:48: error: expected expression before ‘)’ token
     this.friendlist.bind_model (this.friends, fr => new FriendListRow (fr as Tox.Friend));

MainWindow.c:1935:5: error: unknown type name ‘GListStore’
MainWindow.c:1939:5: error: unknown type name ‘GListStore’

/home/travis/build/RicinApp/Ricin/src/MainWindow.vala:135:53: error: expected expression before ‘)’ token
       friend.position = friends.get_n_items ();

MainWindow.c: In function ‘ricin_main_window_instance_init’:
MainWindow.c:3723:2: error: unknown type name ‘GListStore’
```

Possible cause: https://github.com/RicinApp/Ricin/issues/19

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,8 +9,8 @@ install:
  - sudo apt-add-repository ppa:vala-team/ppa -y
  - sudo apt-add-repository ppa:mc3man/trusty-media -y
 
-# Travis requires gtk+ 3.18 to compile.
- - sudo apt-add-repository ppa:ubuntu-desktop/ppa -y
+# Travis requires gtk+ 3.16 to compile.
+ - sudo add-apt-repository ppa:rebuntu16/glade-3.19+-trusty -y
 
  - sudo apt-get update -q
  - sudo apt-get install -y checkinstall build-essential texinfo libgtk-3-dev libglib2.0-dev libgdk-pixbuf2.0-dev libsoup2.4-dev libjson-glib-dev libnotify-bin python3 valac-0.30 libnotify-dev libopus-dev ffmpeg
@@ -66,7 +66,6 @@ before_script:
  - cd ..
 
 script:
- - cd /home/travis/build/RicinApp/Ricin
  - make autogen
  - make install
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -67,7 +67,7 @@ before_script:
 
 script:
  - make autogen
- - make install
+ - sudo make install
 
 notifications:
   email: false

